WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3514,7 +3516,7 @@ e_byzantium = {
 	k_trebizond = {
 		color={ 100 151 33 }
 		capital = 678 # Trapezous
-		culture = greek
+		culture = pontic
```

Comment:
Given some of the rulers of the title, I would say you confuse local culture and the actual holders.